sentry/api/sentry.api

```diff
@@ -1521,14 +1525,15 @@ public class io/sentry/SpanContext : io/sentry/JsonSerializable, io/sentry/JsonU
 	protected field status Lio/sentry/SpanStatus;
 	protected field tags Ljava/util/Map;
 	public fun <init> (Lio/sentry/SpanContext;)V
```

Comment:
Regarding breaking changes to constructors here:
Duplicating the constructors would still break when explicitly passing null as the compiler then doesn't know which version to call. So a cast would be needed to fix. That's why we decided to just modify the constructors and risk breaking stuff this way as they are niche constructors which we expect to not be used by many and it's a rather easy fix to pass a new TracesSamplingDecision(true/false) instead of the boolean.